astropy/tests/figures/helpers.py

```diff
@@ -30,6 +32,7 @@ def decorator(test_function):
                                        style=style,
                                        savefig_kwargs=savefig_kwargs,
                                        **kwargs)
+        @pytest.mark.skipif(not HAS_PYTEST_MPL, reason='pytest-mpl is required for the figure tests')
```

Comment:
I thought about reverting this and use `-m "not mpl_image_compare"` as suggested in https://github.com/matplotlib/pytest-mpl/issues/183#issuecomment-1302733435 but I cannot figure out a sane way to define that in `tox.ini` because we have a few possible flag combos already and CircleCI can theoretically use any of those flags if it wants to.

https://github.com/astropy/astropy/blob/7e80b9fcfc43f1e720d3d3c8b6adcce8f3a47782/tox.ini#L119-L124

Maybe this is still the best option we have unless people have better ideas.